src/ui/imageview/imageview.ios.ts

```diff
@@ -236,7 +236,7 @@ export default class ImageViewIOS<TEvent extends string = ImageViewEvents> exten
     return this.nativeObject.contentMode;
   }
   set imageFillType(value: ImageFillType) {
-    this.nativeObject.contentMode = NativeFillTypeProps[value] || value;
```

Comment:
This would crash if there is no return value from NativeFillTypeProps. We need to make sure first that we got the return value from NativeFillTypeProps. So sample logic could be followed like below;

const mappedValue = NativeFillTypeProps[value];
if (mappedValue) {
    this.nativeObject.contentMode
}